spec/test.rb

```diff
@@ -0,0 +1,95 @@
+# rubocop: disable Layout/LineLength
+require './enums'
+describe Enumerable do
+  describe '#my_each' do
+    it 'iterates upto the number of items in the list and calls block of code if given for every value' do
+      array = [1, 2, 3]
+      expect(array.my_each { |num| print num }).to eql(array.each { |num| print num })
+      expect((1..5).my_each { |value| print value }).to eql(1..5)
+      expect(array.my_each { |num| print num }).not_to eql([1, 2])
+      expect((1..5).my_each { |value| print value }).not_to eql(1..6)
+      last = 0
+      (1..5).my_each { |value| last = value }
+      expect(last).to eql(5)
+      expect(last).not_to eql(6)
```

Comment:
- [ ] Make use of one "it" block for each test expectation. One "it" block should used for one "expect".

Reply:
Thanks for pointing out I will take care of it